packages/datagateway-common/src/detailsPanels/dls/visitDetailsPanel.component.test.tsx

```diff
@@ -1,31 +1,46 @@
-import React from 'react';
-import VisitDetailsPanel from './visitDetailsPanel.component';
+import type { RenderResult } from '@testing-library/react';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import type { UserEvent } from '@testing-library/user-event/dist/types/setup';
+import axios from 'axios';
+import * as React from 'react';
 import { QueryClient, QueryClientProvider } from 'react-query';
-import { mount, ReactWrapper } from 'enzyme';
-import {
-  useInvestigationDetails,
-  useInvestigationSize,
-} from '../../api/investigations';
+import { Provider } from 'react-redux';
+import { combineReducers, createStore } from 'redux';
+import { StateType } from '../../../lib';
 import { Investigation } from '../../app.types';
+import dGCommonReducer from '../../state/reducers/dgcommon.reducer';
+import VisitDetailsPanel from './visitDetailsPanel.component';
 
-jest.mock('../../api/investigations');
-
-describe('Visit details panel component', () => {
-  let rowData: Investigation;
-  const detailsPanelResize = jest.fn();
-
-  const createWrapper = (): ReactWrapper => {
-    return mount(
+function renderComponent({
+  rowData,
+  detailsPanelResize,
+}: {
+  rowData: Investigation;
+  detailsPanelResize?: () => void;
+}): RenderResult {
+  return render(
+    <Provider
+      store={createStore(
```

Comment:
I wouldn't say it's a big deal right now but certainly something to consider as we move further down the line of RTL migration. Since `createStore` is deprecated, Louise figured out a good way of [making use of the newer method](https://github.com/ral-facilities/operationsgateway/blob/aefde7d4e3d9c11b2339bbb58eafcb70eb041330/src/state/store.tsx#L19), `configureStore`, on OperationsGateway that also reduces code repetition. It will be worth adding this at some point but I don't think it's necessary until we decide to properly migrate everything over.

Reply:
Is `configureStore` from redux toolkit compatible with "legacy" redux architecture which is what `datagateway-dataview` is using at the moment?

Reply:
It seems to be, might be worth changing?

Reply:
I personally couldn't say for sure but if you think it is then great!

Change it if you want to, I wouldn't say it's urgent as there's still a lot of RTL migration to do but doing it with `configureStore` will help as we go further down the line

Reply:
I will keep it for now, I mentioned to Louise about the deprecation, and she said the deprecation is more like "redux toolkit is now preferred" rather than it being actually "deprecated".

Reply:
https://github.com/reduxjs/redux/releases/tag/v4.2.0 It is deprecated in this release.

Reply:
Ah okay, it was only because my IDE marked it as deprecated that I knew at all!